test/e2e/utils.py

```diff
@@ -13,39 +13,46 @@
 # limitations under the License.
```

Comment:
suggestion for organizing all of these tests:

/predictors
/explainers
/transformers
/data

Reply:
reorganized !